roadmap.md

```diff
@@ -0,0 +1,53 @@
+# Roadmap
+
+This file provides an overview of the direction this project is heading. The roadmap is organized in steps that focus on a specific theme, for instance, UX.
+
+## Q1 2024
+
+In this phase of the project, the focus is on ad hoc scanners. The expected features are:
+
+**Adhoc Scan​**
+
+It enables users to target particular repositories or entire organizations for scanning purposes. Additionally, users can filter out archived or forked repositories to narrow down the scope of the scan. 
+
+- Targeted repo scan​
+- Targeted org scan​
+- Filter archived repos​
+- Filter forked repos​
+- Custom keyword search​
+
+## Q2 2024
+
+In this phase of the project, the focus is on File Scanner. 
+The expected features are:
+
+
+**File scanner​**
+
+The file scanner helps users understand the content and structure of their files or directories by scanning them thoroughly.
+- Scan directory​
+- Scan individual files
+
+
+
+## Q3 2024
+
+In this phase of the project, the focus is on utilising ML to improve the product. 
```

Comment:
Already we have ML in scope. Highlight that this is enhanced version for improving accuracy.